<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Property</class>
 <widget class="QWidget" name="Property">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Property Image</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Quantity bits on one pixel:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="numBitOnPix">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Size image in bytes:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="sizeImgInByte">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Number used colors:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="numsUseColors">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Hight of image:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="hightImg">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Width of Image:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="formatFile">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Format of Image:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Name file:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="nameFile">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="widthImg">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
